<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodsSelectWidget</class>
 <widget class="QWidget" name="GoodsSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>28</width>
    <height>47</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>288</width>
    <height>470</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="InStockGoodsSelectTitleWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="InStockGoodsSelectListView" name="listView">
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#leftWidget{
        background-color:#fafafa;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>14</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>14</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="lastPageButton">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
        border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
        border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
border-image: url(:/saleImage/btn-up-disable.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextPageButton">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
        border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed{
        border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled{
        border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #ededed;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="pageLabel">
        <property name="styleSheet">
         <string notr="true">font-size:12px;
color:#999999;
font-family:&quot;微软雅黑&quot;;
padding-top:6px;</string>
        </property>
        <property name="text">
         <string>1/1页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fafafa;</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>51</red>
               <green>51</green>
               <blue>51</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>250</red>
               <green>250</green>
               <blue>250</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:16px;
color:#333333;
background:#fafafa;
font-family:&quot;微软雅黑&quot;;
padding-top:-6px;</string>
         </property>
         <property name="text">
          <string>     总金额：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalLabel">
         <property name="styleSheet">
          <string notr="true">font-size:22px;
color:#ED9A6D;
background:#fafafa;
font-family:&quot;微软雅黑&quot;;
padding-top:-12px;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InStockGoodsSelectTitleWidget</class>
   <extends>QWidget</extends>
   <header>../InStock/InStockGoodsSelectTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InStockGoodsSelectListView</class>
   <extends>QListView</extends>
   <header>../InStock/InStockGoodsSelectListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
